src/globals/brand-colour.ts

```diff
@@ -1,14 +1,20 @@
+/**
+ * @see $brandColours in src/globals/variables.scss . This and that should be updated together.
+ *
+ * @todo consider changing values to remove need for comments. Not doing right now to avoid possible semantic
+ * merge conflicts.
+ */
 export type brandColour =
-  | 'primary'
-  | 'secondary'
-  | 'tertiary'
-  | 'brand-1'
-  | 'brand-2'
-  | 'brand-3'
-  | 'brand-4'
-  | 'brand-5'
-  | 'brand-6'
-  | 'black'
+  | 'primary' // blue
+  | 'secondary' // green
+  | 'tertiary' // coral
+  | 'brand-1' // cc-red
+  | 'brand-2' // wgmf-purple
+  | 'brand-3' // gmf-green
+  | 'brand-4' // emf-yellow
+  | 'brand-5' // c4c-orange
+  | 'brand-6' // mhf-turquoise
   | 'white'
+  | 'black'
```

Comment:
just switched order to match variables.scss

Reply:
Makes sense